lib/src/builder/generation/impl_generation.dart

```diff
@@ -570,20 +574,21 @@ class ImplGenerator {
     }
 
     final consumerClassName = _classNameFromNode(node);
-    final accessorsMixinName = '\$$consumerClassName';
+    final accessorsMixinName = consumerClassName.startsWith(generatedPrefix) ?
+        _publicPropsOrStateClassNameFromConsumerClassName(consumerClassName) :
+        '\$$consumerClassName';
```

Comment:
My understanding is that the accessor mixins should be private, but I think both of these are returning values that are prefixed with `$` instead of `_$`. That being said, it appears to work as expected when using the builder locally.. so maybe I'm just missing something.

Reply:
That's true for props/state/abstractprops/abstractstate. This code is just for props/state mixins. For props/state mixins, the accessors mixin is either `$FooPropsMixin` or `FooPropsMixin`.